components/tools/OmeroJava/test/integration/AbstractServerTest.java

```diff
@@ -285,6 +298,155 @@ public void tearDown() throws Exception {
     }
 
     /**
+     * Cast the map of strings (e.g. the ALL_GROUPS_CONTEXT)
```

Comment:
should be able to use something like `{@link #ALL_GROUPS_CONTEXT}` to refer to it in comments